apps/prairielearn/elements/pl-excalidraw/info.json

```diff
@@ -0,0 +1,17 @@
+{
+  "controller": "pl-excalidraw.py",
+  "dependencies": {
+    "nodeModulesScripts": [],
+    "nodeModulesStyles": ["@prairielearn/excalidraw-builds/dist/index.css"],
+    "elementScripts": [],
+    "elementStyles": ["pl-excalidraw.css"]
+  },
+  "dynamicDependencies": {
+    "elementScripts": {
+      "pl-excalidraw-js": "pl-excalidraw.js"
```

Comment:
Why is this a dynamic dependency?

Reply:
Since pl-excalidraw.js is an ESM, I assumed I can only use ESM through dynamicDependencies (which are implemented through importmaps). 

> When importing modules in scripts, if you don't use the [type=importmap](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#importmap) feature, then each module must be imported using a module specifier that is either an absolute or relative URL

Getting this relative URL in JS is simpler with dynamicDependencies